esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -1843,42 +1843,65 @@ static StringBuffer &getDropZoneHost(const char *planeName, IPropertyTree *plane
     return host;
 }
 
+IPropertyTree *CFileSprayEx::getAndValidateDropZone(const char *path, const char *host)
+{
+    Owned<IPropertyTree> plane = findDropZonePlane(path, host, true, isContainerized()); // NB: don't error if missing in bare-metal
+    if (plane)
+    {
+        if (isContainerized())
+            LOG(MCoperatorWarning, "Host/IP %s is used to spray.", host);
```

Comment:
if used in containerized, i.e. if ECL is spraying to an host, not a plane name, then yes, it really should not happen going forward, i.e. they should use the plane name instead.
However, although it would be nice to flag it, so that ultimately feedback could be given to the user, this would be too verbose, and a warning like this probably belongs at the fileservice level so that it could be propagated into the workunit/job warnings that the job/user sees.

I think for now, it would be best to remove this, but leave a comment (at the top of the method), to make it clear it shouldn't be resolving planes by hostnames in containerized but kept for backward compatibility for now.

Reply:
The LOG lines are removed and the comments are added.